environment.yml

```diff
@@ -19,10 +19,15 @@ dependencies:
   - ipython=8.23
   - scipy
   - plotly
-  - festim=2.0a3
-  - plotly
+  - festim=2.0b0
+  - sphinxcontrib-bibtex>=2.6
+  - pybtex-docutils>=1.0.3
+  - ipywidgets
+  - widgetsnbextension
+  - jupyterlab_widgets
   - pip
   - pip:
     - h_transport_materials
+    - pybtex>=0.25
```

Comment:
This isn't needed since we pin down pybtex below, maybe we just keep the `>=` requirement